tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./configs/base.tsconfig",
   "compilerOptions": {
+    "skipLibCheck": false,
```

Comment:
isn't this the default?

Reply:
We enabled in the base tsconfig to speed up the build. I've disabled it in VS Code, so one can write new d.ts files in a type-safe manner.